help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -67,14 +65,16 @@ To connect to this destination, follow the steps described in the [destination c
 
 ### Authenticate to LiveRamp {#authenticate}
 
-To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+To authenticate to the destination, fill in the required fields (to be provided by LiveRamp) and select **[!UICONTROL Connect to destination]**.
 
-![Platform UI image showing the destination connection screen.l](../../assets/catalog/advertising/liveramp-distribution/liveramp-distribution-new-connection.png)
+![Platform UI image showing the destination connection screen.l](../../assets/catalog/advertising/liveramp-distribution/lr_connection_auth.png)
 
+* **[!UICONTROL LiveRamp Organization ID]**: The organization ID of your LiveRamp account (listed as _owner_org_ in your LiveRamp-provided credentials).
+* **[!UICONTROL Password]**: Your LiveRamp account password (listed as _secret_key_ in your LiveRamp-provided credentials).
```

Comment:
Please wrap the LR-provided IDs in code format, so they're easier to follow:

`_owner_org_` etc.